.github/workflows/build-release.yml

```diff
@@ -0,0 +1,72 @@
+---
+
+name: Build release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build release
+    runs-on: [ubuntu-latest]
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SAMLTRACER_BUILD_TOKEN }}
```

Comment:
Secrets are stored here: https://github.com/simplesamlphp/SAML-tracer/settings/secrets/actions
But since they are secret, you can't see their contents.. It's just a personal access token I generated

Reply:
Hm, that confirms my feeling that I am not yet fully authorized here in the project... 😉 When I follow the link, I only get a 404. I can't even access the general settings page.

Who could fix this?